admin/services/nca-tools-user.service.js

```diff
@@ -29,7 +28,8 @@ const service = {
       const mtcRoleName = roleService.mapNcaRoleToMtcRole(ncaUser.UserType, school)
       const role = await roleService.findByTitle(mtcRoleName)
       const user = {
-        identifier: ncaUser.EmailAddress,
+        identifier: ncaUser.externalAuthenticationId,
+        displayName: ncaUser.EmailAddress,
```

Comment:
This doesn't seem quite right.   Shouldn't the display name be a name, e.g. `ncaUser.UserName`?

Reply:
We don't actually care what the email address is, and it sometimes isnt an actual email address! (go figure).

We do however store the contents of that field in the top right of the admin header, so we are accomodating this with the 'displayName' field.

Reply:
btw - in the payload i think its actually called `userName`

Reply:
So, just to be sure, is the displayName okay, or do we want the UserName instead? (The spec was saying "populate users.displayName field with email address").

Reply:
IIRC email address is always populated, username is not.  leave as is for now

Reply:
What kind of auth service doesn't give us the real name?